packages/shared/src/components/cards/SharedPostCardFooter.tsx

```diff
@@ -0,0 +1,37 @@
+import classNames from 'classnames';
+import React, { ReactElement } from 'react';
+import { Post } from '../../graphql/posts';
+
+type SharedPostCardFooterProps = {
+  isShort: boolean;
+} & Pick<Post, 'sharedPost'>;
+
+export const SharedPostCardFooter = ({
+  sharedPost,
+  isShort,
+}: SharedPostCardFooterProps): ReactElement => {
+  return (
+    <div
+      className={classNames(
+        'flex flex-1 gap-2 p-3 mb-2 rounded-12 border border-theme-divider-tertiary',
+        isShort ? 'flex-row items-center' : 'flex-col',
+      )}
+    >
+      <span
+        className={classNames(
+          'typo-footnote text-theme-label-secondary',
+          isShort ? 'line-clamp-4 flex-1' : 'line-clamp-2',
+        )}
+      >
+        {sharedPost.title}
+      </span>
+      <div
+        className={classNames(
+          'rounded-xl',
+          isShort ? 'h-full aspect-square' : 'flex-1',
```

Comment:
What is `isShort` when does it happen? Nothing major but I think we need to come up with a more straightforward description.

Reply:
Left one is short (attached article)
<img width="511" alt="image" src="https://user-images.githubusercontent.com/1488164/214514322-a1c569cf-cc9a-4935-96b7-b813dcac7f7e.png">